Unit-02/01-blueprints/app.py

```diff
@@ -0,0 +1,9 @@
+from project import app
+
+
+if __name__ == '__main__':
+	app.run(debug=True)	
+
```

Comment:
FYI if this was a code review during company projects I'd ask you to remove unnecessary white space at the end of this file (i.e. the last three lines)